docs/engineering/devops.md

```diff
@@ -1,4 +1,4 @@
-# Development Process Overview
```

Comment:
I am not sure "DevOps" is the right word here as it is much broader than the content on this page. I think the original title or something like "Development and Hosting Environments" might be clearer.